packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -165,6 +165,8 @@ export function callUser({userIdOrEmail, constraints = {audio: true, video: true
 
 export function hangupCall(call) {
   return (dispatch) => {
+    // Don't update call states after hangup
+    call.off();
```

Comment:
Added this here because actually calling `hangupCall` would call itself a second time on the `disconnect` event.